src/components/Notifications/NotificationItem.types.tsx

```diff
@@ -0,0 +1,11 @@
+import { NotificationLocale } from "./types"
+import { NotificationItemImageProps } from "./NotificationItemImage.types"
+
+type NotificationItemProps = {
+  image: NotificationItemImageProps
```

Comment:
What if we spread out the `NotificationItemImageProps` so we don't have to repeat the image keyword when defining an image for the notification item?